### ISSUE TITLE
version update and no changelog? sad...

### ISSUE THREAD

**Issue by qianshui423** [eclipsesource/J2V8#505]

**Comment by qianshui423** [comment]
j2v8 how to debug in chrome

**Comment by ahmadov** [comment]
We usually post an article about the new features in the J2V8. There will be a new article about changes and new J2V8 Inspector API!

**Comment by qianshui423** [comment]
> We usually post an article about the new features in the J2V8. There will be a new article about changes and new J2V8 Inspector API!

I have a question: users need to develop debug ui by new J2V8 Inspector API? support chrome devTools remote debug?

**Comment by qianshui423** [comment]
> We usually post an article about the new features in the J2V8. There will be a new article about changes and new J2V8 Inspector API!

'node --inspect' can debug in the chrome devTools. Does J2V8 support the debug plan or already?

**Comment by ahmadov** [comment]
yes, J2V8 version 6.1.0 supports the debugging via V8 Inspector API. see: #478

**Comment by ilyamuromets** [comment]
@ahmadov Hi! How to enable inspector in J2V8 v6.1.0 ? I don't see [https://github.com/eclipsesource/J2V8/commit/fca6a49e9448bcc39356352cd86055a207de19ce](these) changes in my build.

**Comment by mkshiblu** [comment]
Hi, how can I migrate to 6.1.0 from 4.6.0?

I used to do add the platform-specific libs in Gradle for a Java Project like below -

```
implementation group: 'com.eclipsesource.j2v8', name: 'j2v8_win32_x86_64', version: '4.6.0'
```

I tried changing to this -

```
implementation group: 'com.eclipsesource.j2v8', name: 'j2v8', version: '6.1.0'
```

Now in the code, the error says `com.eclipsesource` not found. I could, however, see that **j2v8-6.1.0.aar.asc** is present in the external library section after doing Gradle build. 

I also want to know more about using the latest node.js.

**Comment by josiahbryan** [comment]
@mkshiblu did you ever get any answers...?

**Comment by mkshiblu** [comment]
@josiahbryan No. But I found out that they have stopped building for Java from probably 4.6. You can build yourself for Java Windows from their building documentation. I found a v4.8.5 online which I am using now in my Java project.

**Comment by josiahbryan** [comment]
Oh, okay @mkshiblu - so it's still maintained and updated, just not prebuilt for you?

**Comment by mkshiblu** [comment]
@josiahbryan Yes, exactly.

**Comment by NiuGuohui** [comment]
> We usually post an article about the new features in the J2V8. There will be a new article about changes and new J2V8 Inspector API!

It's been a year. Have you published any related articles?
